.github/workflows/build.yaml

```diff
@@ -84,12 +84,41 @@ jobs:
 
       - name: "Upload"
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          # This is a personal access token for user wojtekmach-erlef-bot that has write access.
+          # We need a token with `workflows` permissions because by pushing git tags
+          # (from upstream, erlang/otp) we're modifying workflow files (.github/workflows)
+          # on those tags and that is restricted. Perhaps this would be not necessary if this repo
+          # was a _fork_ of erlang/otp.
```

Comment:
@starbelly, it should be fairly simple to create an ErlEF bot and then share secrets with certain repositories (e.g. this one) via the Settings, so @wojtekmach doesn't need a PAT. You need to be an admin. or the org. so I guess you can do it. I can walk you through the process as I've done it before, already.

Reply:
It looks like a workflow token was created, I think this is resolved now?

Reply:
It's not working yet. I was trying to workaround permissions issues to prove the idea but no luck yet. If you could set it up I'd appreciate it.

Reply:
That's odd, what you have seems correct,  that is using `GITHUB_TOKEN` and setting the permissions in the workflow file (per https://docs.github.com/en/actions/security-for-github-actions/security-guides/automatic-token-authentication#permissions-for-the-github_token) 

If you know what the issue is, and give pass a pointer, I'm happy to resolve.

Reply:
As I mentioned in the other comment, I'd like to postpone this indefinitely. I'm going to remove the related leftover code from the repo too.